src/api/world.py

```diff
@@ -110,7 +145,7 @@ def get_block(self, x: int, y: int, z: int) -> str:
         offset_x, offset_z = x - 16 * cx, z - 16 * cz
         chunk = self.get_chunk(cx, cz)
         block = chunk[offset_x, y, offset_z].blocks
-        return self._wrapper.mapping_handler[block]
+        return self._wrapper.mapping_handler[block.item()]
```

Comment:
Should probably not use the ".item" function. If the issue is doing something like "isinstance(var, int)" not working, we can create our own "isInt()" function which will check for both a regular int and a numpy int. Using ".item" everywhere would become a mess IMO.

Reply:
I'll look to see if adding that creates any other undefined behaviour, I'm not sure if Python dict/lists properly take in number dtypes like uint8 and such

Reply:
Ok, let me know how it goes. That's the least important issue for me and I don't mind ignoring it, I just prefer it to stay a bit more elegant.